#CREATE request parameters
geometry_msgs/Pose current_pose
geometry_msgs/Pose  target_pose
float64 limit_time

---  
#CREATE response parameters

bool control_ok